app/pipes/city.weather.pipe.ts

```diff
@@ -1,35 +1,36 @@
 import {Pipe, PipeTransform} from '@angular/core';
 import {WeatherService} from '../weather/weather.service';
 import {KelvinToCelsius} from '../pipes/celsius.pipe';
+import {Response} from '@angular/http';
```

Comment:
please, move this import to the top and separate ```@angular``` and other thirdparty imports from custom imports with an empty line